apps/rustadventure/src/pages/index.tsx

```diff
@@ -39,7 +41,9 @@ export default function Home() {
               </h3>
             </div>
             <div className="max-w-sm mx-auto">
-              <ConvertkitSubscribeForm />
+              <ConvertkitSubscribeForm
+                onSubmit={() => router.push('/confirmed')}
```

Comment:
the redirect to `confirmed` happens here. We no longer hit the `confirm` endpoint, though we COULD